### PR TITLE
fix issue where requests to localhost would not contain remotePathPrefix

### PR DESCRIPTION
**Pull request by anthansson**
Fix issue where requests to localhost would not contain remotePathPrefix. For example any requests made to claimsService has a remotePathPrefix = '/claims'. This would not be added as a prefix if running on localhost, hence the actual requests had to have different requests URLs depending on if running on localhost. 

## Optional checklist
- [ ] Unit tests written
- [x] Tested locally
- [x] Codescouted

### CODE REVIEW

File: src/api/upstream.ts [hunk 1]
```diff
@@ -33,7 +33,7 @@ export const createUpstream = (
     const allHeaders = (headers as any) as { [key: string]: string }
 
     const baseUrl: string = config.UPSTREAM_MODE === "local"
-      ? `http://localhost:${localPort}`
+      ? `http://localhost:${localPort}${remotePathPrefix}`
```
Comment (anthansson) [inline src/api/upstream.ts:36]: we want remotePathPrefix no matter what

File: src/api/upstreams/claimService.ts [hunk 2]
```diff
@@ -43,7 +43,7 @@ export enum ClaimOutcomeDto {
 export const createClaimServiceClient = (client: HttpClient): ClaimServiceClient => {
   return {
     getMemberClaims: async () => {
-      const res = await client.get(`/claims/listClaims`)
+      const res = await client.get(``)
```
Comment (anthansson) [inline src/api/upstreams/claimService.ts:46]: Then we don't need prefix here now. The listClaims was removed in claimsService as a get request made to '/claims' is quite intuitive.

Comment (AlexSJ1990) [reply src/api/upstreams/claimService.ts:46]: won't we still need the /claims here?

**Review by AlexSJ1990 — COMMENTED** [review]
Nice! one small question

File: src/api/upstreams/claimService.ts (hunk 2 above)
Comment (anthansson) [reply src/api/upstreams/claimService.ts:46]: No that’s the point of the whole pr. The prefix is automatically added in upstream.ts above :)

Comment (anthansson) [reply src/api/upstreams/claimService.ts:46]: Just happens that ’/listClaims’ also was removed, but separetely in another pr that I haven’t merged yet. So wasn’t sure to leave or remove /listClaims here as well.

Comment (AlexSJ1990) [reply src/api/upstreams/claimService.ts:46]: ahh okay i see! awesome :)

**Review by AlexSJ1990 — APPROVED** [review]
✅

**Pull request merged**